modules/sdk/gradle-plugins/src/main/java/com/liferay/gradle/plugins/NodeDefaultsPlugin.java

```diff
@@ -49,9 +49,6 @@ protected void configureDefaults(Project project, NodePlugin nodePlugin) {
 		return NodePlugin.class;
 	}
 
-	private NodeDefaultsPlugin() {
```

Comment:
Since `NodeDefaultsPlugin` is a real plugin now, you should also remove the `INSTANCE` singleton, and apply it like a real plugin with `GradleUtil.applyPlugin`.